torrent/logging.go

```diff
@@ -6,31 +6,37 @@
  *
  */
 
-package main
+package torrent
 
 import (
 	"log"
 )
 
-func loginfo(fmt string, args ...interface{}) {
+var VERBOSITY uint32 = 0
```

Comment:
Yeah, you're right; I started changing the way I handled verbosity, and then got caught up in something else, and forgot to finish it. Sorry 'bout that.